bigbluebutton-web/grails-app/conf/bigbluebutton.properties

```
@@ -81,6 +81,19 @@ numConversionThreads=5
 #------------------------------------
 numFileProcessorThreads=2
 
+#------------------------------------
+# Number of threads the BBB server uses to fetch the list of plugins
+# at the beginning of a session (/create). Min=1 means a slower fetch response
+# - will delay the entire list causing slower create response, and therefore,
+# slower client join. Max=half the number of cores for very powerful servers
+#------------------------------------
+numPluginManifestsFetchingThreads=5
+
+#------------------------------------
+# Each plugin manifest fetch shouldn't take longer than this time (milliseconds)
+#------------------------------------
+pluginManifestFetchTimeout=3000
```

```suggestion
pluginManifestFetchTimeout=15000
```

It could be a bit more tolerant. If the server experiences a slight delay, it may exceed 3 seconds and fail.